semantic-core/src/Analysis/ImportGraph.hs

```diff
@@ -118,5 +118,9 @@ importGraphAnalysis = Analysis{..}
         string s = pure (Value (String s) mempty)
         asString (Value (String s) _) = pure s
         asString _ = pure mempty
-        record fields = pure (Value Abstract (foldMap (valueGraph . snd) fields))
+        record fields = do
+          for_ fields $ \ (k, v) -> do
+            addr <- alloc k
+            assign addr v
+          pure (Value Abstract (foldMap (valueGraph . snd) fields))
```

Comment:
We were leaving record fields uninitialized in the abstract analyses, which meant that later lookups would fail.